test/dx-clone.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+const config = require('config');
+const client = require('./lib/client').client;
+const nav = require('./lib/nav/navigation')(client, config);
+const micis = require('./lib/auth/micis')(client);
+const should = require('should');
+
+describe('Test data exchange functionality for cloning a request', function() {
+  this.timeout(config.defaultTimeout);
+
+  before('initialize', function(done) {
+    client.clientReady.then(function boot() {
+      if (!micis.loggedOn) {
+        micis.logon();
+      }
+
+      client.call(done);
+    });
+  });
+
+  it('should open Data Exchange', function(done) {
+    nav
+      .micisMenu
+      .clickNested('Browse Available Data')
+      .call(done);
+  });
+
+  it('should verify that [Previous Request Templates] have loaded', function(done) {
+    const previousRequestTemplates = '#requestMenu > option';
```

Comment:
The reason that I did this was because I felt like it made things more readable once you were actually in the client promise chain.  For example on lines 41-50, This seems easier to read:
```js
    const mrFilter = 'div.draggable.filter.MRFilter.ui-draggable';
    const filterContainer = 'div#filterContainer > div.statisticsLabel > div > label';

    client
      .waitForExist(mrFilter)
      .getText(filterContainer)
      .then(response => {
        response.should.be.eql(['MR', 'Assessments', 'Studies', 'Subjects']);
      })
      .call(done);
```
rather than this:
```js
    client
      .waitForExist('div.draggable.filter.MRFilter.ui-draggable')
      .getText('div#filterContainer > div.statisticsLabel > div > label')
      .then(response => {
        response.should.be.eql(['MR', 'Assessments', 'Studies', 'Subjects']);
      })
      .call(done);
```
However, I agree that it is somewhat _non_-idiomatic to turn a `string` into a named variable just for a single use.  I guess I could go either way.  What do you prefer?  I'm happy to change it if you would like.